src/utility.js

```diff
@@ -92,7 +92,22 @@ function sum(x) {
   return x.reduce((a, b) => a + b, 0);
 }
 
-function mergeInto(obj, other) {
+global.LibraryParam = Object.freeze({
+  DisallowOverride: 1,
+});
+
+// if allowOverride is false, it shows error in case of symbol redefinition
+function mergeInto(obj, other, libraryParams = 0) {
```

Comment:
What about using an options object here.. see https://travishorn.com/using-the-options-object-js-pattern-221f083aadbc.

Then this would be `function mergeInto(obj, other, options) {` and that check could be `if ('noOverride' in options) {`?

Reply:
changed to options object, but using noOverride: true, because key in object should have some value. checking 
if (options.noOverride)